apidoc/Titanium/UI/ListView.yml

```diff
@@ -14,9 +14,9 @@ description: |
     </table>
 
     Use the <Titanium.UI.createListView> method or **`<ListView>`** Alloy element to create a `ListView`.
-    
```

Comment:
Please check your Atom settings, the whole PR has those blank lines in it, making it unreadable for reviews. Thx!